### PR TITLE
Unit 3.1: npm and yarn scripts

### PR DESCRIPTION
**Pull request by frankiebarrios**
## What It Does
This pull request is set to complete Unit 3.1 of the ux-onboarding-project.

This work is taking place to complete unit 3.1 of the ux-onboarding-project.

**Implementation:**
- Added script to package.json to run server at http://localhost:1820 using 'yarn start'.
- Updated README with further instructions.

[closes #15 ](https://github.com/frankielb/frankie-barrios-ux-aop/issues/12)

## How To Test
- Install dependencies using instruction in README.
- Start server using yarn start.
- Access site at http://localhost:1820.

## Notes/Caveats

None
## Checklist

Under penalty of public shaming, I avow that I:
**Many items in list N/A** 
- [X] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [X] Verified that there are no compiler or linter errors or warnings
- [X] Verified the build in production(optimized) mode
- [X] Updated the docs, if necessary
- [X] Included the appropriate labels
- [X] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
**Many items in list N/A** 
- [X] Chrome
- [X] Firefox
- [X] Safari
- [X] Edge
- [X] Internet Explorer

## Dependencies
1. Install Node and npm
2. Install Yarn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # UX Team Intern Onboarding Project
 UX Team Intern onboarding project.
-Testing SSH Connection
 # How To Use
-
+Run npm install to install dependencies
+```
+$ npm install
```
Comment (kddrake) [inline README.md]: Should be 
```suggestion
$ yarn install 
```

Using both `npm` and `yarn` can cause problems, as they use slightly different lock file. You'll want to make sure your consistently using `yarn`

**Review by kddrake — CHANGES_REQUESTED** [review]
Can you get `master` merged into this branch so that all your additions you made in unit 2 aren't also being committed, again, here?

File: README.md (hunk 1 above)
Comment (frankiebarrios) [reply README.md]: Got it

**Review by kddrake — CHANGES_REQUESTED** [review]
Derp, forgot to say you should remove `package-lock.json`, run `yarn install`, an commit the `yarn.lock` file

File: package.json [hunk 2]
```diff
@@ -2,9 +2,11 @@
   "name": "frankie-barrios-ux-iop-1",
   "version": "1.0.0",
   "description": "UX Team Intern Onboarding Project",
-  "main": "index.js",
+  "main": "index.html",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (kddrake) [inline package.json]: now that we have an actual script to run, lets remove this

File: package.json [hunk 3]
```diff
@@ -2,9 +2,11 @@
...
+    "say-hello": "echo 'Hello World'",
```
Comment (kddrake) [inline package.json]: now that we have an actual script to run, lets remove this, too

**Review by kddrake — APPROVED** [review]

**Review by kddrake — CHANGES_REQUESTED** [review]
I'm unable to actually hit `http://localhost:1820`, but was able to hit the provided urls given by `http-server`. May want to ensure we can use `https`... I'm guessing that's why I can't use localhost

**Review by kddrake — APPROVED** [review]

**Pull request merged**